arbnode/batch_poster.go

```diff
@@ -414,13 +418,50 @@ func (s *batchSegments) CloseAndGetBytes() ([]byte, error) {
 }
 
 func (b *BatchPoster) encodeAddBatch(seqNum *big.Int, prevMsgNum arbutil.MessageIndex, newMsgNum arbutil.MessageIndex, message []byte, delayedMsg uint64) ([]byte, error) {
-	method, ok := b.seqInboxABI.Methods["addSequencerL2BatchFromOrigin0"]
+	methodName := "addSequencerL2BatchFromOrigin0"
+	if b.config().EIP4844 {
+		methodName = "addSequencerL2BatchWithBlobs0"
```

Comment:
You shouldn't 0 postfix this method. We 0 postfixed the fromorigin method because there were two solidity methods with the same name